addons/common/functions/fnc_registerItemReplacement.sqf

```diff
@@ -20,8 +20,12 @@
 params [["_oldItem", "", [0,""]], ["_newItems", "", ["", []]], ["_replaceInherited", false, [false]]];
 TRACE_3("registerItemReplacement",_oldItem,_newItems,_replaceInherited);
 
+
+// Setup on first run
+if (isNil QGVAR(itemReplacements)) then {
+
 // CBA player event handler function
-private _fnc_replaceItems = {
+DFUNC(registerItemReplacement_replaceItems) = compileFinal toString {
```

Comment:
I didn't tab it over so the diff is readable, but can shift it over if it looks good

Reply:
LGTM.